trestle/core/commands/partial_object_validate.py

```diff
@@ -60,40 +59,23 @@ def _run(self, args: argparse.Namespace) -> int:
             log.set_log_level_from_args(args)
             file_path: pathlib.Path = args.file.resolve()
             if not file_path.exists() or not file_path.is_file():
-                logger.error('File path provided does not exist or is a directory')
-                return CmdReturnCodes.COMMAND_ERROR.value
+                raise TrestleError('File path provided does not exist or is a directory')
+
             element_str: str = args.element
-            if ',' in element_str:
-                logger.error('Only a single element path is allowed.')
```

Comment:
is this check happening somewhere?  It should give warning.

Reply:
Changed to warning